lite/examples/sound_classification/android/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/MainActivity.kt

```diff
@@ -18,84 +18,151 @@ package org.tensorflow.lite.examples.soundclassifier
 
 import android.Manifest
 import android.content.pm.PackageManager
+import android.media.AudioRecord
 import android.os.Build
 import android.os.Bundle
+import android.os.Handler
+import android.os.HandlerThread
 import android.util.Log
 import android.view.WindowManager
 import androidx.annotation.RequiresApi
 import androidx.appcompat.app.AppCompatActivity
 import androidx.core.content.ContextCompat
+import androidx.core.os.HandlerCompat
 import org.tensorflow.lite.examples.soundclassifier.databinding.ActivityMainBinding
+import org.tensorflow.lite.task.audio.classifier.AudioClassifier
+import java.util.*
```

Comment:
Nit: no wildcard imports (per Java style guide)

Reply:
Unfortunately this is the default setting of import optimization in Android Studio and I haven't found how to disable it :(

Given it's a minor issue, I'll leave it for now.